src/blox/stochastic.jl

```diff
@@ -22,12 +22,35 @@ mutable struct OUBlox <: NeuralMassBlox
         sts = @variables x(t)=0.0 [output=true] jcn(t) [input=true]
         @brownian w
 
-        eqs = [D(x) ~ -(x-μ)/τ + jcn + sqrt(2/τ)*σ*w]
+        eqs = [D(x) ~ (-x + μ + jcn)/τ + sqrt(2/τ)*σ*w]
         sys = System(eqs, t; name=name)
         new(namespace, true, sys)
     end
 end
 
+function get_ts_data(t, dt::Real, data::Array{Float64})
+    idx = ceil(Int, t / dt)
+
+    return idx > 0 ? data[idx] : 0.0
```

Comment:
```suggestion
    return ifelse(idx > 0, data[idx], 0.0)
```
use this instead of the ternary operator in symbolic expressions

Reply:
Oh my, I did not want to commit but to comment.... anyhow.

Reply:
the thing with `ifelse` is that it evaluates every expression whereas the ternary operator only evaluates the relevant one based on the condition. Thus I spare myself the little index "hack" and it works with the symbolic expression/function it seems. Or at least I didn't get any error?

Reply:
This is my source 
https://docs.sciml.ai/ModelingToolkit/stable/basics/FAQ/#How-do-I-handle-if-statements-in-my-symbolic-forms?

> ifelse is that it evaluates every expression

that is a good thing for AD IIRC. In any case I think you can restructure the function to use `ifelse` with two branches that won't break on evaluation.